BuildingSync.xsd

```diff
@@ -11463,6 +11472,7 @@
         </xs:annotation>
         <xs:simpleType>
           <xs:restriction base="xs:string">
+
```

Comment:
@markborkum can you remove this extra line?

Reply:
@nllong, No. Not because I don't want to. It's not appearing my text editor (Atom).

Reply:
Hmm, weird, okay. Not sure where this is coming from then...